pkg/store/proxy.go

```diff
@@ -348,6 +374,12 @@ func startStreamSeriesSet(
 		defer wg.Done()
 		defer close(s.recvCh)
 
+		i := 0
```

Comment:
I feel like a more descriptive name like `responses` would be better. I am way too used to seeing `i` as a placeholder variable in for-loops for the index :smile: Here we do an infinite loop so it probably doesn't fit as well.

Reply:
agreed